pandas/core/indexes/multi.py

```diff
@@ -243,10 +243,18 @@ def __new__(cls, levels=None, codes=None, sortorder=None, names=None,
 
         if verify_integrity:
             result._verify_integrity()
+
         if _set_identity:
             result._reset_identity()
+
         return result
 
+    def _validate_codes(cls, level, code):
```

Comment:
can you make this accept self, and add doc-string

Reply:
changed and doc-string added